libs/core/src/lib/text/text.component.html

```diff
@@ -10,6 +10,7 @@
     </span>
 
     <a fd-link
+       tabindex="0"
```

Comment:
`tabindex` makes a link focusable but does not allow us to trigger click using `enter` key down.
Instead, we could add `href="#"` attribute that would make a link focusable and react on `enter` by default but we should prevent the default behaviour on click

Reply:
Or keep `tabindex="0"` and listen to `keydown eneter` event as well

Reply:
Modified. Can you please re review.